File: SlowlifeMobile (2)/SlowlifeMobile/test/src/com/mycompany/entities/evenement/Evenement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities.evenement;

import java.util.Date;

/**
 *
 * @author Ahmed Ezzine
 */
public class Evenement {
     private long idEvent ;
    private String nom ;
    private String type ;
    private String dateDeb ;
    private String dateFin ;
    private String lieu ;
    private String description ;
    private int idUser;
    private String nomC;
    private String prenomC;
    private int participants;
    
    public Evenement() {
        super();
    }

    public Evenement(long idEvent, String nom, String type, String dateDeb, String dateFin, String lieu, String description, int idUser) {
        this.idEvent = idEvent;
        this.nom = nom;
        this.type = type;
        this.dateDeb = dateDeb;
        this.dateFin = dateFin;
        this.lieu = lieu;
        this.description = description;
        this.idUser = idUser;
    }

    public Evenement(String nom, int participants) {
        this.nom = nom;
        this.participants = participants;
    }

   /* public Evenement(String toString, String toString0, String toString1, String valueOf, Date valueOf0, Date valueOf1) {
        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }*/
    
    

    public long getIdEvent() {
        return idEvent;
    }

    public void setIdEvent(long idEvent) {
        this.idEvent = idEvent;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getDateDeb() {
        return dateDeb;
    }

    public void setDateDeb(String dateDeb) {
        this.dateDeb = dateDeb;
    }

    public String getDateFin() {
        return dateFin;
    }

    public void setDateFin(String dateFin) {
        this.dateFin = dateFin;
    }

    public String getLieu() {
        return lieu;
    }

    public void setLieu(String lieu) {
        this.lieu = lieu;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getIdUser() {
        return idUser;
    }

    public void setIdUser(int idUser) {
        this.idUser = idUser;
    }

    public void setNomC(String nomC) {
        this.nomC = nomC;
    }

    public void setPrenomC(String prenomC) {
        this.prenomC = prenomC;
    }
    
    public String getNomCoach () {
        return nomC + " " + prenomC;
    }

    public int getParticipants() {
        return participants;
    }

    public void setParticipants(int participants) {
        this.participants = participants;
    }
    
    
    
    @Override
    public String toString() {
        
       return "Event{" + "Nom=" + nom + ", Type=" + type + ", description=" + description +""
                + "Date Debut=" + dateDeb + "Date Fin="+ dateFin +"nbr_participants= " +participants 
                +"Region= " +lieu +'}';
    }
    
}

File: src/Entity/Evenement.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Evenement
 *
 * @ORM\Table(name="evenement", indexes={@ORM\Index(name="fk_userEvent", columns={"idUser"})})
 * @ORM\Entity
 */
class Evenement
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=70, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="lieu", type="string", length=150, nullable=false)
     */
    private $lieu;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=50, nullable=false)
     */
    private $type;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut", type="date", nullable=false)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_fin", type="date", nullable=false)
     */
    private $dateFin;

    /**
     * @var int
     *
     * @ORM\Column(name="idUser", type="integer", nullable=false)
     */
    private $iduser;


    private  $nomC;
    private $prenomC;
    private $participants;

    /**
     * Evenement constructor.
     * @param int $id
     * @param string $nom
     * @param string $lieu
     * @param string $type
     * @param string $description
     * @param \DateTime $dateDebut
     * @param \DateTime $dateFin
     * @param int $iduser
     */
    public function Event__construct(int $id, string $nom, string $lieu, string $type, string $description, \DateTime $dateDebut, \DateTime $dateFin, int $iduser)
    {
        $this->id = $id;
        $this->nom = $nom;
        $this->lieu = $lieu;
        $this->type = $type;
        $this->description = $description;
        $this->dateDebut = $dateDebut;
        $this->dateFin = $dateFin;
        $this->iduser = $iduser;
    }

    /**
     * Evenement constructor.
     * @param string $nom
     * @param $participants
     */
    public function participant__construct(string $nom, $participants)
    {
        $this->nom = $nom;
        $this->participants = $participants;
    }



    /**
     * @return int
     */
    public function getId(): int
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId(int $id): void
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getNom(): string
    {
        return $this->nom;
    }

    /**
     * @param string $nom
     */
    public function setNom(string $nom): void
    {
        $this->nom = $nom;
    }

    /**
     * @return string
     */
    public function getLieu(): string
    {
        return $this->lieu;
    }

    /**
     * @param string $lieu
     */
    public function setLieu(string $lieu): void
    {
        $this->lieu = $lieu;
    }

    /**
     * @return string
     */
    public function getType(): string
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType(string $type): void
    {
        $this->type = $type;
    }

    /**
     * @return string
     */
    public function getDescription(): string
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription(string $description): void
    {
        $this->description = $description;
    }

    /**
     * @return \DateTime
     */
    public function getDateDebut(): \DateTime
    {
        return $this->dateDebut;
    }

    /**
     * @param \DateTime $dateDebut
     */
    public function setDateDebut(\DateTime $dateDebut): void
    {
        $this->dateDebut = $dateDebut;
    }

    /**
     * @return \DateTime
     */
    public function getDateFin(): \DateTime
    {
        return $this->dateFin;
    }

    /**
     * @param \DateTime $dateFin
     */
    public function setDateFin(\DateTime $dateFin): void
    {
        $this->dateFin = $dateFin;
    }

    /**
     * @return int
     */
    public function getIduser(): int
    {
        return $this->iduser;
    }

    /**
     * @param int $iduser
     */
    public function setIduser(int $iduser): void
    {
        $this->iduser = $iduser;
    }

    /**
     * @return mixed
     */
    public function getNomC()
    {
        return $this->nomC;
    }

    /**
     * @param mixed $nomC
     */
    public function setNomC($nomC): void
    {
        $this->nomC = $nomC;
    }

    /**
     * @return mixed
     */
    public function getPrenomC()
    {
        return $this->prenomC;
    }

    /**
     * @param mixed $prenomC
     */
    public function setPrenomC($prenomC): void
    {
        $this->prenomC = $prenomC;
    }

    /**
     * @return mixed
     */
    public function getParticipants()
    {
        return $this->participants;
    }

    /**
     * @param mixed $participants
     */
    public function setParticipants($participants): void
    {
        $this->participants = $participants;
    }




}
